### PR TITLE
azure-pim-cli 0.7.0 (new formula)

### PR DESCRIPTION
**Pull request by tnaumann**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

NOTES:
- There _are_ outdated pull requests for this formula.
- Fails audit because of `* GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)`

In other words, this pull request can be closed because of audit failure but I would like to include in case anyone else in the community is looking for it.

### CODE REVIEW

**Comment by tnaumann** [comment]
Closing since this is not a sufficiently popular tool to be included
